src/makeSchema.ts

```diff
@@ -64,13 +65,61 @@ generateSchema.withArtifacts = async (
   schemaTypes: string
   tsTypes: string
 }> => {
-  const { schema, missingTypes, finalConfig } = makeSchemaInternal(config)
+  const { schema, missingTypes, finalConfig, hasSDLDirectives } = makeSchemaInternal(config)
   const typegenConfig = resolveTypegenConfig(finalConfig)
   const { schemaTypes, tsTypes } = await new TypegenMetadata(typegenConfig).generateArtifactContents(
     schema,
-    typeFilePath
+    typeFilePath,
+    hasSDLDirectives
   )
   assertNoMissingTypes(schema, missingTypes)
   runAbstractTypeRuntimeChecks(schema, finalConfig.features)
   return { schema, schemaTypes, tsTypes }
 }
+
+/** Builds the schema, we may return more than just the schema from this one day. */
+export function makeSchemaInternal(config: SchemaConfig) {
```

Comment:
Do we want to export that anymore?

Reply:
Yeah I think I'm using it in the tests, I don't believe it's publicly exported on the module